src/System.Security.Cryptography.Cng/tests/SymmetricCngTestHelpers.cs

```diff
@@ -271,6 +273,22 @@ internal static bool SupportsPersistedSymmetricKeys
             }
         }
 
+        private static bool? s_isAdministrator;
+
+        internal static bool IsAdministrator
+        {
+            get
+            {
+                if (!s_isAdministrator.HasValue)
+                {
+                    s_isAdministrator =
+                        new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator);
```

Comment:
Rewrote this to use Lazy instead of Nullable; and to put the WindowsIdentity in a using statement.

Also used an expression-bodied member to dereference the Value property.  Not sure if I feel dirty, or like I'm moving with the times :smile:.